src/zelos/api/zelos_api.py

```diff
@@ -637,6 +637,12 @@ def thread(self) -> Thread:
     def main_binary(self) -> Optional[ParsedBinary]:
         """
         Returns the parsed main binary, if it exists, otherwise returns None.
+        Note that the "main" binary denotes the binary that is loaded by
```

Comment:
You may want to reword the sentence to leave room for situations like your run_dll capability. I imagine that would have a main_binary different than the target_binary?

Reply:
Good idea.